### PR TITLE
753 redirection to home page

### PR DESCRIPTION
**Pull request by kacperpONS**
# Motivation and Context

Added back button to uac already used, uac inactive and too many request pages.

# What has changed

Added back button
Updated the tests for checking for the back button on those pages
Added tests for uac inactive page
Added the word "Back" to the translation as it wasn't there 

# How to test?

Run the updated tests.
Test manually with uac already used and uac deactivated.

# Links

https://trello.com/c/ZrtuQafN/753-redirection-to-home-page-required-from-access-code-has-already-been-used-page-3

# Screenshots (if appropriate):

### CODE REVIEW

File: app/start_handlers.py [hunk 1]
```diff
@@ -97,14 +97,3 @@ def _display_missing_uac_error(request, display_region):
...
-
-@start_routes.view(r'/' + View.valid_display_regions + '/' + user_journey + '/exit/')
-class StartExit(View):
-    async def get(self, request):
-        display_region = request.match_info['display_region']
-        self.log_entry(request, display_region + '/' + user_journey + '/exit')
-        await invalidate(request)
-        return HTTPFound(
-            request.app.router['Start:get'].url_for(display_region=display_region)
-        )
```
Comment (ryangrundy7) [inline app/start_handlers.py:110]: Seems unrelated to your changes, why is this being removed?

**Review by ryangrundy7 — APPROVED** [review]

**Review by gavinleeedwards — APPROVED** [review]
All good 👍
